packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -103,14 +105,15 @@ contract OPStackManager is ISemver {
     string public constant version = "1.0.0-beta.3";
 
     /// @notice Address of the SuperchainConfig contract shared by all chains.
-    SuperchainConfig public immutable superchainConfig;
+    SuperchainConfig public superchainConfig;
 
     /// @notice Address of the ProtocolVersions contract shared by all chains.
-    ProtocolVersions public immutable protocolVersions;
+    ProtocolVersions public protocolVersions;
```

Comment:
These can stay as immutables, because they won't change, and if we do need to change them we can do so by upgrading the implementation

Reply:
Done in latest version.